common/sgx/endorsements.c

```diff
@@ -379,30 +400,17 @@ oe_result_t oe_get_sgx_endorsements(
         "Failed to get intermediate certificate. %s",
         oe_result_str(result));
 
-    //
-    // Get revocation information
-    //
-    OE_CHECK_MSG(
-        oe_get_revocation_info_from_certs(
-            &leaf_cert, &intermediate_cert, &revocation_info),
-        "Failed to get certificate revocation information. %s",
-        oe_result_str(result));
-
-    //
-    // Get QE identify info
-    //
+    // Get quote verification collateral
     OE_CHECK_MSG(
-        oe_get_qe_identity_info(&qe_id_info),
-        "Failed to get quote enclave identity information. %s",
+        oe_get_sgx_quote_verification_collateral_from_certs(
+            &leaf_cert, &quote_verification_collateral),
+        "Failed to get certificate quote verification collateral information. "
+        "%s",
         oe_result_str(result));
 
-    //
-    // Create endorsement structure
-    //
     OE_CHECK_MSG(
         oe_create_sgx_endorsements(
```

Comment:
It would be interesting to see how long does this function take.  Could you please find out?

Reply:
8.5 microseconds